### PR TITLE
github: fix quoting in github workflow for jitter tests

### PR DESCRIPTION
**Pull request by xnox**
Nested quoting got ignore previously. And this way one can specify
string name directly.

Successfully run with Jitter at
https://github.com/xnox/openssl/actions/runs/10223149419/job/28289017013

### CODE REVIEW

**Comment by xnox** [comment]
@t8m fix for workflow as mentioned in https://github.com/openssl/openssl/pull/24844

File: .github/workflows/run-checker-daily.yml [hunk 1]
```diff
@@ -207,7 +207,7 @@ jobs:
     - name: checkout fuzz/corpora submodule
       run: git submodule update --init --depth 1 fuzz/corpora
     - name: config
-      run: ./config --with-rand-seed=none enable-jitter --with-jitter-include=jitter/ --with-jitter-lib=jitter/ -DOPENSSL_DEFAULT_SEED_SRC='"JITTER"' && perl configdata.pm --dump
+      run: ./config --with-rand-seed=none enable-jitter --with-jitter-include=jitter/ --with-jitter-lib=jitter/ -DOPENSSL_DEFAULT_SEED_SRC='\"JITTER\"' && perl configdata.pm --dump
```
Comment (nhorman) [inline .github/workflows/run-checker-daily.yml:210]: Given that users will have to define this as well, might it be better to allow OPENSSL_DEFAULT_SEED_SRC to be defined without quotes, and just stringify it in the code? i.e., in rand_lib.c:

```
#define STR(x) #x
...
name = STR(OPENSSL_DEFAULT_SEED_SRC);
```

Comment (xnox) [reply .github/workflows/run-checker-daily.yml:210]: Ok will fix it to not require quotes on the argument to configure.

Comment (xnox) [reply .github/workflows/run-checker-daily.yml]: Used existing `OPENSSL_MSTR` define for this

**Comment by slontis** [comment]
Should the 2 pod files be updated also?

**Comment by xnox** [comment]
> Should the 2 pod files be updated also?

done

**Review by nhorman — APPROVED** [review]

**Review by tom-cosgrove-arm — APPROVED** [review]
LGTM

**Review by t8m — APPROVED** [review]

**Comment by t8m** [comment]
@nhorman @tom-cosgrove-arm if you agree with urgent, please merge this. IMO it is CI fix -> urgent.

**Comment by nhorman** [comment]
Ok with urgent. Can merge shortly

**Comment by nhorman** [comment]
merged, thank you!

**Comment by nhorman** [comment]
merged, thank you!

**Comment by tom-cosgrove-arm** [comment]
Sorry, have been AFK - okay with urgent, FWIW!

**Pull request closed without merging**